src/pages/Trade.vue

```diff
@@ -62,4 +90,9 @@ export default defineComponent({
     @apply md:mt-8;
   }
 }
+
+.widget-card {
+  width: 260px;
+  max-width: 260px;
```

Comment:
the `max-width` seems redundant?

Reply:
the animations behave weirdly without this